clients/python/dd_client.py

```diff
@@ -124,154 +62,60 @@ def __init__(self,host="localhost",port=8080,proto=0,apiversion="0.1"):
         self.__host = host
         self.__port = port
         self.__proto = proto
-        self.__returntype=self.RETURN_PYTHON
+        self.__returntype = self.RETURN_JSON
         if proto == self.__HTTP:
-            self.__ddurl='http://%s:%d'%(host,port)
+            self.__ddurl = 'http://%s:%d' % (host, port)
         else:
-            self.__ddurl='https://%s:%d'%(host,port)
-
+            self.__ddurl = 'https://%s:%d' % (host, port)
 
-    def set_return_format(self,f):
-        assert f == self.RETURN_PYTHON or f == self.RETURN_JSON or f == self.RETURN_NONE
+    def set_return_format(self, f):
+        assert f == self.RETURN_JSON or f == self.RETURN_TEXT or f == self.RETURN_NONE
         self.__returntype = f
 
-    def __return_format(self,js):
-        if self.__returntype == self.RETURN_PYTHON:
-            return json.loads(js.decode('utf-8'))
-        elif self.__returntype == self.RETURN_JSON:
-            return js
+    def __return_data(self, r):
+        if self.__returntype == self.RETURN_JSON:
+            return r.json()
```

Comment:
Requests makes educated guesses about the encoding of the response based on the HTTP headers, so it will work with other encodings. More here **[response-content](http://docs.python-requests.org/en/master/user/quickstart/#response-content)**